lib/bats-core/tracing.bash

```diff
@@ -50,7 +50,7 @@ bats_print_stack_trace() {
 			# don't print "from function `source'"",
 			# when failing in free code during `source $test_file` from bats-exec-file
 			! [[ "$fn" == 'source' &&  $index -eq $count ]]; then 
-			printf "from function \`%s' " "$fn"
+			printf "from function '%s' " "$fn"
```

Comment:
I am not sure why ` was used in the first place but changing that is going to require some rework at least in our test suite. Can you motivate why this change is necessary?

Reply:
Yep, I reworked the test suite to support the ' instead of the `. It's actually been there since day one and I'm almost certain it was a mistake: https://github.com/sstephenson/bats/commit/c850527cce7134f4adf4fe6dac07214678deb72b#diff-2285302ec7310b89fd9edfef3694598bebfa056fdbfd7e5d607026f64a68d734R80

Reply:
Reverted -- I'll do a separate PR for this